### PR TITLE
ci: add taplo

### PR DESCRIPTION
**Pull request by KSXGitHub**
* Add `taplo` to CI.
* Add `allow_blank_lines = 1` to taplo config.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -93,3 +93,10 @@ jobs:
           restore-cache: false
 
       - run: cargo fmt --all -- --check
+
+      - name: Install Taplo CLI
+        uses: ./.github/actions/binstall
+        with:
+          packages: taplo@0.8.1
```
Comment (KSXGitHub) [inline .github/workflows/ci.yml:100]:
```suggestion
          packages: taplo-cli@0.8.1
```

File: crates/fs/Cargo.toml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+name                 = "pacquet-fs"
+description          = "Filesystem utility functions used by pacquet"
+version              = "0.0.1"
+publish              = false
+authors.workspace    = true
+edition.workspace    = true
+homepage.workspace   = true
+keywords.workspace   = true
+license.workspace    = true
+repository.workspace = true
```
Comment (zkochan) [inline crates/fs/Cargo.toml:12]: I don't really like this alignment with fixed width before =. It means that if there is a new field with a longer length, the all the other unrelated line will be change too, making diffs bigger.

**Review by zkochan — APPROVED** [review]

File: crates/fs/Cargo.toml (hunk 2 above)
Comment (KSXGitHub) [reply crates/fs/Cargo.toml:12]: @zkochan I tried removing `align_width = true` from `.taplo.toml` before making this PR. The `Cargo.toml` files look messy without column alignment, so I decided to keep it as it is.
